src/ts/components/Sidebar.tsx

```diff
@@ -0,0 +1,65 @@
+import { h, Fragment } from "preact";
+import Store from ".././state/store";
+import Results from "./Results";
+import Controls from "./Controls";
+import { Commands } from ".././commands";
+import { IMatch } from ".././interfaces/IMatch";
+import { MatcherService } from "..";
+import { useState } from "preact/hooks";
+
+interface IProps {
+    store: Store<IMatch>;
+    matcherService: MatcherService<IMatch>;
+    commands: Commands;
+    contactHref?: string;
+    feedbackHref?: string;
+    active: boolean;
+}
+
+const Sidebar = ({
+    store,
+    matcherService,
+    commands,
+    contactHref,
+    feedbackHref,
+    active
+}: IProps) => {
+const [isActive, setIsActive] = useState(active);
+   return (
+   <div className="Sidebar__section">
+    {isActive ? ( 
+        <Fragment>
+            <Controls
+                store={store}
+                setDebugState={commands.setDebugState}
+                setRequestOnDocModified={commands.setRequestOnDocModified}
+                requestMatchesForDocument={commands.requestMatchesForDocument}
+                fetchCategories={matcherService.fetchCategories}
+                getCurrentCategories={matcherService.getCurrentCategories}
+                addCategory={matcherService.addCategory}
+                removeCategory={matcherService.removeCategory}
+                feedbackHref={feedbackHref}
+                deactivate={() => setIsActive(false)}
+            />
+            <Results
+                store={store}
+                applySuggestions={commands.applySuggestions}
+                applyAutoFixableSuggestions={commands.applyAutoFixableSuggestions}
+                selectMatch={commands.selectMatch}
+                indicateHover={commands.indicateHover}
+                stopHover={commands.stopHover}
+                contactHref={contactHref}
+            />
+    </Fragment>
+        
+    ) : (
+        <button
+        type="button"
+        className="Button"
+        onClick={() => setIsActive(true)}
+    >Open Typerighter</button>
```

Comment:
I think it would be nice if we make the switched off typerighter even less intrusive. Possibly similar to the Figma desing: https://www.figma.com/proto/mvbwYPVo3JCgQaP5APV0OK/Typerighter?node-id=42%3A1&scaling=min-zoom

Reply:
👍 working on that!